pkg/resourcemanager/mysql/mysqlhelper.go

```diff
@@ -31,43 +31,8 @@ const DriverName = "mysql"
 // assume will exist).
 const SystemDatabase = "mysql"
 
-func GetMySQLAADResourceID() string {
-	// TODO: Switch this to use config.Environment().ResourceIdentifiers.OSSRDBMS
-	// TODO: when that change is in azure-go-sdk
-	// TODO: See: https://github.com/Azure/go-autorest/pull/635
-	envName := config.Environment().Name
-
-	if envName == "AzureUSGovernmentCloud" {
-		return "https://ossrdbms-aad.database.usgovcloudapi.net"
-	} else if envName == "AzureChinaCloud" {
-		return "https://ossrdbms-aad.database.chinacloudapi.cn"
-	} else if envName == "AzureGermanCloud" {
-		return "https://ossrdbms-aad.database.cloudapi.de"
-	}
-
-	return "https://ossrdbms-aad.database.windows.net"
-}
-
-func GetMySQLDatabaseDNSSuffix() string {
-	// TODO: We need an environment specific way of getting the DNS suffix
-	// TODO: which the Go SDK doesn't seem to have.
-	// TODO: see: https://github.com/Azure/azure-sdk-for-go/issues/13749
-	// TODO: In the meantime we'll fabricate our own
-	envName := config.Environment().Name
-
-	if envName == "AzureUSGovernmentCloud" {
-		return "mysql.database.usgovcloudapi.net"
-	} else if envName == "AzureChinaCloud" {
-		return "mysql.database.chinacloudapi.cn"
-	} else if envName == "AzureGermanCloud" {
-		return "mysql.database.cloudapi.de"
-	}
-
-	return "mysql.database.azure.com"
-}
-
 func GetFullSQLServerName(serverName string) string {
-	return serverName + "." + GetMySQLDatabaseDNSSuffix()
+	return serverName + "." + config.Environment().MySQLDatabaseDNSSuffix
```

Comment:
Good idea to move this to configuration.

Reply:
Making sure I understand: Are you saying that the change made here is good, _or_ that I should move this whole `GetFullSQLServerName` method to `config`?

I _think_ you're saying the first thing but just wanted to confirm.

Reply:
I was confirming that _the change you'd already done_ was a good one, moving knowledge of the DNS suffix for MySQL Databases out of our code to where it can be customized without a recompilation.

Reply:
Thanks for the confirmation